index.bs

```diff
@@ -307,35 +307,36 @@ The {{XR}} object has a <dfn>pending immersive session</dfn> boolean, which MUST
 When the <dfn method for="XR">requestSession(|mode|)</dfn> method is invoked, the user agent MUST run the following steps:
 
   1. Let |promise| be [=a new Promise=].
+  1. Let |immersive| be <code>true</code> if |mode| is {{XRSessionMode/"immersive-vr"}} or {{XRSessionMode/"immersive-ar"}}, and <code>false</code> otherwise.
+  1. If |immersive| is <code>true</code>:
+    1. If the algorithm is not [=triggered by user activation=], [=reject=] |promise| with a "{{SecurityError}}" {{DOMException}} and return |promise|.
+    1. If [=pending immersive session=] is <code>true</code> or [=active immersive session=] is not <code>null</code>, [=reject=] |promise| with an "{{InvalidStateError}}" {{DOMException}} and return |promise|.
+  1. Set [=pending immersive session=] to be <code>true</code>.
```

Comment:
This line appears to need a bit of additional indentation to logically fall under the `If immersive is true:` condition.